modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/freelist/AbstractFreeList.java

```diff
@@ -379,11 +379,11 @@ public AbstractFreeList(
         long loadSize = 0;
 
         for (int b = BUCKETS - 2; b > 0; b--) {
-            long bsize = pageSize - ((REUSE_BUCKET - b) << shift);
+            long freeSize = b << shift;
```

Comment:
First summand ('pageSize') is lost here. 
Does it mean loadSize was calculated incorrectly before this fix?

Reply:
I think it's basically the same computation, only hairier.

Reply:
Could you add Javadoc to shift field? This will allow to easier understand code of free list.